docs/components/modeler/about-modeler.md

```diff
@@ -7,29 +7,29 @@ keywords: ["process mapping tool"]
 
 <span class="badge badge--cloud">Camunda Platform 8 only</span>
 
-Any executable process needs a **BPMN diagram** designed and configured beforehand.
+Any executable process needs a **[Business Process Model and Notation (BPMN)](./bpmn/bpmn.md) diagram** designed and configured beforehand.
 
 A BPMN diagram is used to visually outline the structure and flow of a process. As a result, the process can be more easily understood by various stakeholders.
 
 In tandem, different events and implementation details (such as the conditions within a gateway or the specifications of a service task) must be configured in the model so the workflow engine understands what must be executed once the process reaches a certain task.
 
-Camunda offers a few tools to design your diagrams and implement them:
+Camunda offers two tools to design and implement your diagrams:
 
 - [Web Modeler](./web-modeler/launch-cloud-modeler.md)
 - [Desktop Modeler](./desktop-modeler/index.md)
 
-Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally, all while integrating your local development environment.
+Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.
 
 :::note
-Interested in editing the underlying XML of BPMN and DMN diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
+Interested in editing the underlying XML of BPMN and [Decision Model and Notation (DMN)](./dmn/dmn.md) diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.
```

Comment:
Why do we link only one of both tools here? Let's link both for consistency.

Reply:
I usually link on first reference, so BPMN is linked on line 10. Does this feel awkward to you?

Reply:
I often dive into individual sections of a documentation page (and don't assume this is read top -> bottom). Hence I'd consistently link both here. 

Only linking first time is what I know works from documents that are read strictly top to bottom.

Reply:
Adjusted in latest commit 👍